<launch>
	<arg name="quad" default="SQ01s"/>
	<arg name="x" default="0" />
	<arg name="y" default="0" />
	<arg name="z" default="1.0" />
	<arg name="yaw" default="0.0" />
	
	<!-- Start mader commands -->
	<node ns="$(arg quad)" name="mader_commands" pkg="mader" type="mader_commands.py" output="screen"/>

	<!-- Start Perfect tracker -->
	<include file="$(find mader)/launch/perfect_tracker_and_sim.launch">
  		<arg name="quad" value="$(arg quad)"/>
  		<arg name="gazebo" value="false"/>
        <arg name="x" value="$(arg x)"/>
        <arg name="y" value="$(arg y)"/>
        <arg name="z" value="$(arg z)"/>
        <arg name="yaw" default="$(arg yaw)"/>
        <!-- <arg name="perfect_tracker" default="false"/> -->
 	 </include> 


</launch>
